src/security/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright(c) 2006 to 2019 ADLINK Technology Limited and others
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v. 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License
+# v. 1.0 which is available at
+# http://www.eclipse.org/org/documents/edl-v10.php.
+#
+# SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+#
+cmake_minimum_required(VERSION 3.7)
+
+add_subdirectory(api)
+add_subdirectory(core)
+add_subdirectory(builtin_plugins)
```

Comment:
The plugin and core should not be build when ```ENABLE_SSL``` option is ```off```.

Reply:
done